aya/src/programs/mod.rs

```diff
@@ -533,3 +579,24 @@ impl_try_from_program!(
     CgroupSkb,
     LircMode2
 );
+
+pub struct ProgramInfo(bpf_prog_info);
```

Comment:
Can you add some docs for the struct and the accessors please?

Reply:
I've added some rustdoc (a bit rudimentary)